e2e/tests/user/registration-ui.spec.ts

```diff
@@ -90,14 +88,7 @@ describe('User registration tests:', () => {
     const nextButton = await createAccountPage.getNextButton();
     expect(await nextButton.isCursorNotAllowed()).toEqual(true);
 
-    const institutionSelect = new SelectComponent(page, 'Select your institution');
-    await institutionSelect.select(INSTITUTION_VALUE.BROAD);
-
-    const emailAddress = await Textbox.forLabel(page, {textContains: FIELD_LABEL.INSTITUTION_EMAIL, ancestorNodeLevel: 2});
-    await emailAddress.type(configs.broadInstitutionEmail);
-
-    const roleSelect = new SelectComponent(page, 'describes your role');
-    await roleSelect.select(INSTITUTION_ROLE_VALUE.UNDERGRADUATE_STUDENT);
+    await createAccountPage.fillOutInstitution();
 
     await nextButton.waitUntilEnabled();
```

Comment:
nit: there are two different conventions on this line and the next for async functions. I would have expected to see `await nextButton.enabled();` for consistency.